src/list.json

```diff
@@ -140794,5 +140794,19 @@
     "country": "Taiwan",
     "id": 10032,
     "abbr": "NUTC"
+  },
+  {
+    "name": "Alex Ekwueme Federal University Ndufu-Alike",
+    "domains": [
+      "funai.edu.ng"
+    ],
+    "web_pages": [
+      "https://funai.edu.ng"
+    ],
+    "country": "Nigeria",
+    "alpha_two_code": "NG",
+    "state-province": null,
+    "id": 10033,
+    "abbr": "aefunai"
```

Comment:
Thanks for the PR! Would you please confirm if this is the correct abbreviation of your university? The abbreviation will be **_displayed as-is without converting to upper case letters_**.

Reply:
Yes, that is the correct abbr, only that it is lowercase.
Let me make them uppercase.
The most i saw were lowercases so i thought that was a convention.

Reply:
looked again and it seems like a convention.

Reply:
Thanks a lot! It is not a convention; it is just that I generated the initial values from the domain names. Most of those abbreviations are initialized with generated/inferred values. That is why most of them are lower cased. Humans overwrites are welcomed over best-guess inference.